src/app_service/service/community/service.nim

```diff
@@ -1318,27 +1318,30 @@ QtObject:
 
 
   proc asyncCommunityInfoLoaded*(self: Service, communityIdAndRpcResponse: string) {.slot.} =
-    let rpcResponseObj = communityIdAndRpcResponse.parseJson
-    if (rpcResponseObj{"response"}{"error"}.kind != JNull):
-      let error = Json.decode($rpcResponseObj["response"]["error"], RpcError)
-      error "Error requesting community info", msg = error.message
-      return
+    try:
+      let rpcResponseObj = communityIdAndRpcResponse.parseJson
+      if (rpcResponseObj{"response"}{"error"}.kind != JNull):
```

Comment:
I suppose that instead of catching the exception we could fix the logic here for correct processing of response. We're trying to access `rpcResponseObj{"response"}` even when there's no such field, because an error occurred in the async task. 

https://github.com/status-im/status-desktop/blob/6e5f33ba495776bb7fa561b28a37b930c7987826/src/app_service/service/community/async_tasks.nim#L32-L47

Also, with such solution we won't print the actual error from status-go 🙁

Reply:
I agreed :) can we call it `hot fix`? Cos I can't moving forward without it :( you can reassign this issue to me if you not started yet and can do it (cc: @jrainville)